exercises/concept/library-fees/.docs/hints.md

```diff
@@ -14,7 +14,7 @@
 - There is a [built-in function][naive-date-time-to-time] that changes a `NaiveDateTime` struct to a `Time` struct.
 - There is a [built-in function][time-compare] that checks which one of two `Time` structs is bigger.
 
-## 3. Calculate the return datetime
+## 3. Calculate the return date
```

Comment:
Hint wasn't showing up because of heading mismatch between hints and instructions